.vsts-ci.yml

```diff
@@ -2,8 +2,23 @@ resources:
 - repo: self
 
 jobs:
+- job: HasRuntimeChanged
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+  - script: |
+      echo "Determine if runtime changed."
+      git diff --quiet origin/master -- src/rt || echo "##vso[task.setvariable variable=testRuntime;isOutput=true]On" #set variable to testRuntime to On
```

Comment:
This could be simplified with:
```
    if git diff...; then
      ... turn RT tests on
    else
      .. turn RT tests off
    fi
```
as Paul suggested.